cpp/server/ct-server.cc

```diff
@@ -222,10 +231,30 @@ int main(int argc, char* argv[]) {
   // For now, run with a dedicated thread pool as the executor for our
   // consistent store to avoid the possibility of DoS through thread starvation
   // via HTTP.
-  ThreadPool etcd_pool(2);
-  EtcdConsistentStore<LoggedCertificate> consistent_store(&etcd_pool,
+  ThreadPool internal_pool(4);
+  EtcdConsistentStore<LoggedCertificate> consistent_store(&internal_pool,
                                                           &etcd_client,
                                                           "/root", node_id);
+  // Put a sensible single-node config into FakeEtcd. For a real clustered log
+  // we'd expect a ClusterConfig already to be present within etcd as part of
+  // the provisioning of the log.
+  {
+    ct::ClusterConfig config;
+    config.set_minimum_serving_nodes(1);
+    config.set_minimum_serving_fraction(1);
+    LOG(INFO) << "Setting default single-node ClusterConfig:\n"
+              << config.DebugString();
+    consistent_store.SetClusterConfig(config);
```

Comment:
Buries defaults for the temporary-testing-only-known-to-be-a-single-node-instance where we *can* come up with a guaranteed good set of configs, vs. the risk that someone doesn't bother/realise they forgot to configure their For Realz cluster to their specific number of nodes and serving requirements because it starts and appears to work just fine...

What's a sensible default value for `minimum_num_serving_nodes` that works for clusters of 2, 5, and 50 nodes?

An ASCII proto buf fed into a config tool is more or less what I had in mind, and I'm not averse to shipping an example_config.txt which contains some values which would probably result in a functioning (if not optimal) cluster, but I want the log operator to have to consciously take the step of configuring the cluster during the the process of provisioning it.

Comments in the .proto file (and our as yet vapourware administration docs!) explaining in detail the knobs and levers is definitely a good thing to have - I'll add something there now, but I suspect that we'll later want to go over everything in there and make sure all the explanations fit together properly.